profile/README.md

```diff
@@ -1,12 +1,10 @@
 ## Welcome 👋
 
-<!--
+# Where you are 📍
+This is the GitHub organization of the [Amt für Daten und Statistik Basel-Landschaft](https://www.baselland.ch/politik-und-behorden/direktionen/finanz-und-kirchendirektion/daten-statistik). Here, we publish code and documents used in our daily work that may also be of interest to the community and external partners.
```

Comment:
Office for Data and Statistics BL instead of Amt für Daten und Statistik Basel-Landschaft?